resources/migrate-to-swift.md

```diff
@@ -53,13 +53,34 @@ Once the previous command is complete, run `pod install` or `pod update` to upda
 
 ### Swift Package Manager
 
-You can now also use Swift Package Manager (SPM) to manage your Adobe Experience Platform Mobile SDK dependencies. To add the AEP SDK Packages to your application, from the Xcode 13 menu select:
+#### Swift Package Collection
+
+In Swift 5.5, the Swift Package Manager (SPM) adds support for [package collections](https://www.swift.org/blog/package-collections). You can configure package collection in Xcode 13 for easy installation of AEP SDKs. The Swift package collection for the Adobe Experience Platform SDKs is available at the [Adobe Open Source site](https://opensource.adobe.com/aepsdk-core-ios/swift/packages/aep.json).
+
+To add the Swift package collection in Xcode 13, select File followed by Add Packages", selecting the plus sign on the bottom left and choosing "Add Swift Package Collection"
+
+![](../.gitbook/assets/spm-add-package-collection.png)
```

Comment:
Just more as a heads up for future consistence with the Platform docs, we generally use rounded edged rectangles and use the colour #19E641 when we want to bring attention to an item.